### PR TITLE
Add default HTML, CSS, and JS files, use updated bramble.addNewFile()

### PR DESCRIPTION
**Pull request by humphd**
Watching my user testing videos, it became clear that when adding new files, providing something to start with would be nice vs. a totally blank page.  Depends on https://github.com/humphd/brackets/pull/444

### CODE REVIEW

**Comment by humphd** [comment]
@flukeout, @hannahkane: can I get you two to review the 3 default files here (html.txt, js.txt, css.txt)?  I want to provide *something* when you make a new file vs. just a blank page, and ideally I want it to be a mix of a) helpful; b) to-the-point; c) providing key info for getting more help.

Let me know if you'd change these to better achieve those goals.

File: public/scripts/editor/js/fc/bramble-menus.js [hunk 1]
```diff
@@ -84,19 +84,18 @@ define(function(require) {
...
-      bramble.addNewFile(type);
-    }
-
-    function downloadFileToFilesystem(location, localPath, callback) {
+    function downloadFileToFilesystem(location, fileOptions, callback) {
       callback = callback || function noop() {};
 
+      menu.close();
+
       return $.get(location)
```
Comment (gideonthomas) [inline public/scripts/editor/js/fc/bramble-menus.js]: We don't need to return here right?

**Comment by gideonthomas** [comment]
r=me with confirmation about the default html file contents

**Comment by hannahkane** [comment]
We talked about this here: https://github.com/mozilla/thimble.webmaker.org/issues/541

**Comment by humphd** [comment]
Thanks for reminding me of that ticket, @hannahkane.  I've done an update to the default files to make them more like what was discussed in #541.  Here's what it looks like when you add each one:

screenshot 2015-08-26 13 58 02

screenshot 2015-08-26 13 58 23

screenshot 2015-08-26 13 58 44

NOTE: this is not our default content, it's just what you get when you add a new HTML, CSS, or JS file via the `+` menu.

**Comment by hannahkane** [comment]
copy in comments looks good. I think this addresses feedback we've heard from users that beginning coders don't always know how to reference stylesheets, for example.

**Pull request merged**